src/components/sectionList/listActions/DeleteAction.tsx

```diff
@@ -93,10 +93,8 @@ function ConfirmationDialog({
         { success: true }
     )
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const errorReports = (deleteModelMutation.error as any)?.details?.response
-        ?.errorReports
-
+    const errorReports =
+        deleteModelMutation.error?.details.response.errorReports
```

Comment:
We might want to have type-guard for usage of the error-object. Like isErrorReports(), because this isn't exactly "safe", if we get something unexpected from the server. That's why I think having a component to handle this would be great, but probably something we could add later?

Reply:
> a component to handle this would be great

Agreed!

> this isn't exactly "safe", if we get something unexpected from the server

That's why I added the "optional chaining" operator after every property, just to ensure that this won't throw in production. Maybe add them again until we have a component that handles that?